src/main/java/com/google/cloud/hadoop/io/bigquery/hive/HiveBigQueryMetaHook.java

```diff
@@ -0,0 +1,108 @@
+package com.google.cloud.hadoop.io.bigquery.hive;
+
+import com.google.cloud.hadoop.io.bigquery.BigQueryConfiguration;
+import com.google.common.base.Strings;
+import org.apache.hadoop.hive.metastore.HiveMetaHook;
+import org.apache.hadoop.hive.metastore.api.MetaException;
+import org.apache.hadoop.hive.metastore.api.Table;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+/**
+ *  Class {@link HiveBigQueryMetaHook} can be used to validate and perform different actions during creation and
+ *  dropping of Hive Table.
+ */
+public class HiveBigQueryMetaHook implements HiveMetaHook {
+
+    private static Logger LOGGER = LoggerFactory.getLogger(HiveBigQueryMetaHook.class);
```

Comment:
The common practice in the project is to name loggers LOG

Reply:
Changed the logger name to LOG